recipes/pyrost/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "pyrost" %}
+{% set version = "0.1.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7f289093c3b7f8b58bb5a2ecb499275f78bf3704ee9b5b347b2072c9df156ddb
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux]
+  host:
+    - libblas
+    - python
+    - pip
+    - cython
+    - gsl
+    - h5py
+    - numpy
+    - scipy
+  run:
+    - libblas
```

Comment:
Only add `libblas` to the `host` section, it is automatically added to `run`: https://conda-forge.org/docs/maintainer/knowledge_base.html#blas